Auto update failing for ncspot-mpris

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [ncspot-mpris](https://github.com/termux/termux-packages/blob/26a62b0f936c3672ca61e40831d86909f133d405/x11-packages/ncspot-mpris) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating ncspot-mpris [Current version: 1.0.0]
dpkg: warning: version '.1.1.0' has bad syntax: version number does not start with digit
INFO: package being updated to .1.1.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  249k    0  249k    0     0  1314k      0 --:--:-- --:--:-- --:--:-- 1314k

You are about to commit these changes:

--------------------
diff --git a/x11-packages/ncspot-mpris/build.sh b/x11-packages/ncspot-mpris/build.sh
index 72cd04c0a..373b0b47d 100644
--- a/x11-packages/ncspot-mpris/build.sh
+++ b/x11-packages/ncspot-mpris/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/hrkfdn/ncspot
 TERMUX_PKG_DESCRIPTION="An ncurses Spotify client written in Rust (with MPRIS support)"
 TERMUX_PKG_LICENSE="BSD 2-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.0.0"
+TERMUX_PKG_VERSION=".1.1.0"
 TERMUX_PKG_SRCURL=https://github.com/hrkfdn/ncspot/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=516663b62b9536cb18e6d8eb69470a5b6560f2890e010e8a3d2e8cfc65df9497
+TERMUX_PKG_SHA256=2f22250a4ed7d71657918a98275ac72464bb1be29777a0bd849bb7a013a9e695
 TERMUX_PKG_DEPENDS="dbus, pulseaudio"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(x11/ncspot-mpris): .1.1.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package ncspot-mpris exists in x11 repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building ncspot-mpris for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  87815      0 --:--:-- --:--:-- --:--:-- 88140
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  16759      0 --:--:-- --:--:-- --:--:-- 17000
gpg: Signature made Tue 05 Mar 2024 12:16:13 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  37085      0 --:--:-- --:--:-- --:--:-- 37100
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5032      0 --:--:-- --:--:-- --:--:--  5048
gpg: Signature made Sat 02 Mar 2024 09:43:23 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0   180k      0 --:--:-- --:--:-- --:--:--  183k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  14585      0 --:--:-- --:--:-- --:--:-- 14614
gpg: Signature made Sun 03 Mar 2024 01:01:22 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libexpat@2.6.1 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2023.2 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.4 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.16.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.16.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.16.1 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.7 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dbus@1.15.6-1 if necessary...
Found dbus in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dbus to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-3 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-3 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-3 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-3 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency coreutils@9.4 if necessary...
Found coreutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting coreutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.60.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.1.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2023.12.12 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2023.12.12 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.2.1-1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.2.1-1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.6.0-1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.6.0-1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dash@0.5.12 if necessary...
Found dash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dialog@1.3-20230209-0 if necessary...
Found dialog in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dialog to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency diffutils@3.10 if necessary...
Found diffutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting diffutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency findutils@4.9.0-2 if necessary...
Found findutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting findutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmpfr@4.2.1 if necessary...
Found libmpfr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmpfr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gawk@5.3.0 if necessary...
Found gawk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gawk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.43 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.43 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency grep@3.11 if necessary...
Found grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gzip@1.13 if necessary...
Found gzip in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gzip to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency less@633-1 if necessary...
Found less in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting less to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency procps@3.3.17-5 if necessary...
Found procps in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting procps to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency psmisc@23.6-2 if necessary...
Found psmisc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting psmisc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sed@4.9-1 if necessary...
Found sed in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sed to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-glob@0.6-2 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tar@1.35 if necessary...
Found tar in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tar to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-am@0.8.0 if necessary...
Found termux-am in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am to /home/builder/.termux-build/_cache-all...
Downloading dependency termux-am-socket@1.5.0 if necessary...
Found termux-am-socket in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am-socket to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-exec@1:1.0 if necessary...
Found termux-exec in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-exec to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.39.3-4 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.39.3-4 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.39.3-4 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.39.3-4 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.39.3-4 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.4 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.39.3-4 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.39.3-4 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.39.3-4 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.39.3-4 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.39.3-4 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.0-1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.0-1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-tools@1.40.7 if necessary...
Found termux-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-tools to /home/builder/.termux-build/_cache-all...
Downloading dependency bash@5.2.26 if necessary...
Found bash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtool@2.4.7-1 if necessary...
Found libtool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libtool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libltdl@2.4.7-1 if necessary...
Found libltdl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libltdl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.4-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.23 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.44.2 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.44.2 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.13 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.43 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.2 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.5 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.13 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.8 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.8 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.8 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency glib@2.78.4 if necessary...
Found glib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency glib-cross@2.78.4 if necessary...
Found glib-cross in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib-cross to /home/builder/.termux-build/_cache-all...
Downloading dependency glib-bin@2.78.4 if necessary...
Found glib-bin in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib-bin to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pkg-config@0.29.2-2 if necessary...
Found pkg-config in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pkg-config to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency perl@5.38.2 if necessary...
Found perl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting perl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency texinfo@7.1 if necessary...
Found texinfo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting texinfo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency check@0.15.2-2 if necessary...
Found check in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting check to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-execinfo@0.1-1 if necessary...
Found libandroid-execinfo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-execinfo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libogg@1.3.5 if necessary...
Found libogg in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libogg to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libflac@1.4.3 if necessary...
Found libflac in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libflac to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency flac@1.4.3 if necessary...
Found flac in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting flac to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libopus@1.4 if necessary...
Found libopus in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libopus to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libvorbis@1.3.7-1 if necessary...
Found libvorbis in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libvorbis to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsndfile@1.2.2 if necessary...
Found libsndfile in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsndfile to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsoxr@0.1.3-5 if necessary...
Found libsoxr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsoxr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwebrtc-audio-processing@0.3.1-2 if necessary...
Found libwebrtc-audio-processing in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwebrtc-audio-processing to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency speexdsp@1.2.1 if necessary...
Found speexdsp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting speexdsp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pulseaudio@17.0-1 if necessary...
Found pulseaudio in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pulseaudio to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pulseaudio-glib@17.0-1 if necessary...
Found pulseaudio-glib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pulseaudio-glib to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/hrkfdn/ncspot/archive/v.1.1.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  249k    0  249k    0     0  1529k      0 --:--:-- --:--:-- --:--:-- 1529k
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.76.0-x86_64-unknown-linux-gnu'
info: default toolchain set to '1.76.0-x86_64-unknown-linux-gnu'

  1.76.0-x86_64-unknown-linux-gnu unchanged - rustc 1.76.0 (07dca489a 2024-02-04)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: component 'rust-std' for target 'aarch64-linux-android' is up to date
  Installing ncspot v1.1.0 (/home/builder/.termux-build/ncspot-mpris/src)
    Updating crates.io index
 Downloading crates ...
  Downloaded async-recursion v1.0.5
  Downloaded event-listener-strategy v0.5.0
  Downloaded notify-rust v4.10.0
  Downloaded tracing-attributes v0.1.27
  Downloaded toml_edit v0.19.15
  Downloaded zvariant_utils v1.1.0
  Downloaded event-listener-strategy v0.4.0
  Downloaded async-fs v1.6.0
  Downloaded zbus_macros v4.1.2
  Downloaded zvariant_derive v3.15.1
  Downloaded event-listener v4.0.3
  Downloaded zbus_names v2.6.1
  Downloaded zbus v3.15.1
  Downloaded async-io v1.13.0
  Downloaded rustix v0.37.27
  Downloaded serde_repr v0.1.18
  Downloaded futures-lite v2.2.0
  Downloaded proc-macro-crate v3.1.0
  Downloaded blocking v1.5.1
  Downloaded socket2 v0.4.10
  Downloaded zvariant v3.15.1
  Downloaded zvariant_derive v4.0.2
  Downloaded futures-lite v1.13.0
  Downloaded zbus_names v3.0.0
  Downloaded enumflags2_derive v0.7.9
  Downloaded zvariant v4.0.2
  Downloaded event-listener v5.2.0
  Downloaded async-task v4.7.0
  Downloaded endi v1.1.0
  Downloaded xdg-home v1.1.0
  Downloaded concurrent-queue v2.4.0
  Downloaded event-listener v2.5.3
  Downloaded zbus_macros v3.15.1
  Downloaded async-lock v3.3.0
  Downloaded zbus v4.1.2
  Downloaded enumflags2 v0.7.9
  Downloaded winnow v0.5.40
  Downloaded static_assertions v1.1.0
  Downloaded cfg_aliases v0.1.1
  Downloaded waker-fn v1.1.1
  Downloaded atomic-waker v1.1.2
  Downloaded memoffset v0.9.0
  Downloaded polling v2.8.0
  Downloaded async-channel v2.2.0
  Downloaded proc-macro-crate v1.3.1
  Downloaded hex v0.4.3
  Downloaded piper v0.2.1
  Downloaded derivative v2.2.0
  Downloaded nix v0.28.0
  Downloaded fastrand v1.9.0
  Downloaded async-broadcast v0.5.1
  Downloaded linux-raw-sys v0.3.8
  Downloaded async-lock v2.8.0
  Downloaded toml_edit v0.21.1
  Downloaded io-lifetimes v1.0.11
  Downloaded parking v2.2.0
  Downloaded ordered-stream v0.2.0
  Downloaded async-executor v1.8.0
  Downloaded async-broadcast v0.7.0
   Compiling autocfg v1.1.0
   Compiling proc-macro2 v1.0.78
   Compiling unicode-ident v1.0.12
   Compiling libc v0.2.153
   Compiling cfg-if v1.0.0
   Compiling version_check v0.9.4
   Compiling cc v1.0.89
   Compiling pin-project-lite v0.2.13
   Compiling memchr v2.7.1
   Compiling quote v1.0.35
   Compiling syn v2.0.52
   Compiling typenum v1.17.0
   Compiling libm v0.2.8
   Compiling generic-array v0.14.7
   Compiling num-traits v0.2.18
   Compiling once_cell v1.19.0
   Compiling serde v1.0.197
   Compiling futures-core v0.3.30
   Compiling itoa v1.0.10
   Compiling getrandom v0.2.12
   Compiling syn v1.0.109
   Compiling crossbeam-utils v0.8.19
   Compiling log v0.4.21
   Compiling equivalent v1.0.1
   Compiling futures-io v0.3.30
   Compiling hashbrown v0.14.3
   Compiling indexmap v2.2.5
   Compiling slab v0.4.9
   Compiling pkg-config v0.3.30
   Compiling backtrace v0.3.69
   Compiling adler v1.0.2
   Compiling gimli v0.28.1
   Compiling futures-sink v0.3.30
   Compiling bytes v1.5.0
   Compiling miniz_oxide v0.7.2
   Compiling tracing-core v0.1.32
   Compiling object v0.32.2
   Compiling serde_derive v1.0.197
   Compiling tracing-attributes v0.1.27
   Compiling addr2line v0.21.0
   Compiling lock_api v0.4.11
   Compiling rustc-demangle v0.1.23
   Compiling smallvec v1.13.1
   Compiling parking_lot_core v0.9.9
   Compiling fixedbitset v0.4.2
   Compiling petgraph v0.6.4
   Compiling tracing v0.1.40
   Compiling signal-hook-registry v1.4.1
   Compiling thread-id v4.2.1
   Compiling scopeguard v1.2.0
   Compiling futures-macro v0.3.30
   Compiling futures-channel v0.3.30
   Compiling num-integer v0.1.46
   Compiling parking v2.2.0
   Compiling pin-utils v0.1.0
   Compiling tinyvec_macros v0.1.1
   Compiling winnow v0.5.40
   Compiling toml_datetime v0.6.5
   Compiling futures-task v0.3.30
   Compiling futures-util v0.3.30
   Compiling tinyvec v1.6.0
   Compiling parking_lot v0.12.1
   Compiling tokio-macros v2.2.0
   Compiling concurrent-queue v2.4.0
   Compiling socket2 v0.5.6
   Compiling num_cpus v1.16.0
   Compiling mio v0.8.11
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling cpufeatures v0.2.12
   Compiling percent-encoding v2.3.1
   Compiling tokio v1.36.0
   Compiling fnv v1.0.7
   Compiling protobuf v2.28.0
   Compiling http v0.2.12
   Compiling digest v0.10.7
   Compiling unicode-normalization v0.1.23
   Compiling serde_json v1.0.114
   Compiling unicode-bidi v0.3.15
   Compiling ryu v1.0.17
   Compiling byteorder v1.5.0
   Compiling form_urlencoded v1.2.1
   Compiling zvariant_utils v1.1.0
   Compiling rand_core v0.6.4
   Compiling indexmap v1.9.3
   Compiling bitflags v2.4.2
   Compiling ppv-lite86 v0.2.17
   Compiling base64 v0.21.7
   Compiling rand_chacha v0.3.1
   Compiling idna v0.5.0
   Compiling time-core v0.1.2
   Compiling thiserror v1.0.57
   Compiling hashbrown v0.12.3
   Compiling tokio-util v0.7.10
   Compiling semver v1.0.22
   Compiling powerfmt v0.2.0
   Compiling vcpkg v0.2.15
   Compiling num-conv v0.1.0
   Compiling httparse v1.8.0
   Compiling iana-time-zone v0.1.60
   Compiling chrono v0.4.34
   Compiling time-macros v0.2.17
   Compiling openssl-sys v0.9.101
   Compiling deranged v0.3.11
   Compiling url v2.5.0
   Compiling rand v0.8.5
   Compiling sha1 v0.10.6
   Compiling thiserror-impl v1.0.57
   Compiling ring v0.17.8
   Compiling fastrand v2.0.1
   Compiling num_threads v0.1.7
   Compiling httpdate v1.0.3
   Compiling try-lock v0.2.5
   Compiling want v0.3.1
   Compiling time v0.3.34
   Compiling protobuf-codegen v2.28.0
   Compiling vergen v3.2.0
   Compiling h2 v0.3.24
   Compiling http-body v0.4.6
   Compiling event-listener v5.2.0
   Compiling enumflags2_derive v0.7.9
   Compiling cipher v0.2.5
   Compiling errno v0.3.8
   Compiling tower-service v0.3.2
   Compiling opaque-debug v0.3.1
   Compiling ident_case v1.0.1
   Compiling bitflags v1.3.2
   Compiling darling_core v0.20.8
   Compiling aho-corasick v1.1.2
   Compiling enumflags2 v0.7.9
   Compiling event-listener-strategy v0.5.0
   Compiling rustc_version v0.4.0
   Compiling hyper v0.14.28
   Compiling protobuf-codegen-pure v2.28.0
   Compiling toml_edit v0.19.15
   Compiling event-listener v4.0.3
   Compiling libpulse-sys v1.21.0
   Compiling digest v0.9.0
   Compiling untrusted v0.9.0
   Compiling subtle v2.4.1
   Compiling static_assertions v1.1.0
   Compiling mime v0.3.17
   Compiling glob v0.3.1
   Compiling rustversion v1.0.14
   Compiling regex-syntax v0.8.2
   Compiling async-trait v0.1.77
   Compiling spin v0.9.8
   Compiling librespot-protocol v0.4.2
   Compiling crypto-mac v0.11.1
   Compiling proc-macro-crate v1.3.1
   Compiling event-listener-strategy v0.4.0
   Compiling darling_macro v0.20.8
   Compiling regex-automata v0.4.6
   Compiling aes-soft v0.6.4
   Compiling headers-core v0.2.0
   Compiling futures-executor v0.3.30
   Compiling toml_edit v0.21.1
   Compiling num-iter v0.1.44
   Compiling cookie v0.17.0
   Compiling num-bigint v0.4.4
   Compiling num-rational v0.4.1
   Compiling priority-queue v1.4.0
   Compiling foreign-types-shared v0.1.1
   Compiling openssl v0.10.64
   Compiling io-lifetimes v1.0.11
   Compiling event-listener v2.5.3
   Compiling foreign-types v0.3.2
   Compiling regex v1.10.3
   Compiling futures v0.3.30
   Compiling proc-macro-crate v3.1.0
   Compiling headers v0.3.9
   Compiling librespot-core v0.4.2
   Compiling darling v0.20.8
   Compiling async-lock v3.3.0
   Compiling hmac v0.11.0
   Compiling num-derive v0.3.3
   Compiling openssl-macros v0.1.1
   Compiling block-buffer v0.9.0
   Compiling futures-lite v2.2.0
   Compiling ahash v0.8.11
   Compiling num-bigint v0.2.6
   Compiling polling v2.8.0
   Compiling memoffset v0.7.1
   Compiling atomic-waker v1.1.2
   Compiling native-tls v0.2.11
   Compiling rustix v0.38.31
   Compiling fastrand v1.9.0
   Compiling rustix v0.37.27
   Compiling rustls v0.21.10
   Compiling waker-fn v1.1.1
   Compiling heck v0.4.1
   Compiling async-task v4.7.0
   Compiling futures-lite v1.13.0
   Compiling piper v0.2.1
   Compiling sha-1 v0.9.8
   Compiling pbkdf2 v0.8.0
   Compiling enumset_derive v0.8.1
   Compiling hyper-proxy v0.9.1
   Compiling async-lock v2.8.0
   Compiling aes v0.6.0
   Compiling zvariant_derive v3.15.1
   Compiling rustls-webpki v0.101.7
   Compiling sct v0.7.1
   Compiling async-channel v2.2.0
   Compiling tokio-stream v0.1.14
   Compiling shannon v0.2.0
   Compiling idna v0.3.0
   Compiling enum-map-derive v0.17.0
   Compiling libpulse-simple-sys v1.21.1
   Compiling num-complex v0.4.5
   Compiling uuid v1.7.0
   Compiling android_system_properties v0.1.5
   Compiling async-io v1.13.0
   Compiling async-fs v1.6.0
   Compiling memoffset v0.9.0
   Compiling num-complex v0.2.4
   Compiling num-rational v0.2.4
   Compiling linux-raw-sys v0.4.13
   Compiling openssl-probe v0.1.5
   Compiling utf8parse v0.2.1
   Compiling base64 v0.13.1
   Compiling cfg_aliases v0.1.1
   Compiling linux-raw-sys v0.3.8
   Compiling stable_deref_trait v1.2.0
   Compiling zerocopy v0.7.32
   Compiling lazy_static v1.4.0
   Compiling owning_ref v0.4.1
   Compiling nix v0.28.0
   Compiling anstyle-parse v0.2.3
   Compiling num v0.4.1
   Compiling enum-map v2.7.3
   Compiling cookie_store v0.20.0
   Compiling blocking v1.5.1
   Compiling zvariant v3.15.1
   Compiling enumset v1.1.3
   Compiling strum_macros v0.25.3
   Compiling zvariant_derive v4.0.2
   Compiling ctr v0.6.0
   Compiling derivative v2.2.0
   Compiling serde_repr v0.1.18
   Compiling crossbeam-channel v0.5.12
   Compiling ordered-stream v0.2.0
   Compiling socket2 v0.4.10
   Compiling xdg-home v1.1.0
   Compiling colorchoice v1.0.0
   Compiling unicode-width v0.1.11
   Compiling android-tzdata v0.1.1
   Compiling anstyle-query v1.0.2
   Compiling hex v0.4.3
   Compiling signal-hook v0.3.17
   Compiling unicode-segmentation v1.11.0
   Compiling anstyle v1.0.6
   Compiling webpki-roots v0.25.4
   Compiling endi v1.1.0
   Compiling xi-unicode v0.3.0
   Compiling cursive_core v0.3.7
   Compiling zvariant v4.0.2
   Compiling anstream v0.6.13
   Compiling ureq v2.9.1
   Compiling strum v0.25.0
   Compiling aes-ctr v0.6.0
   Compiling nix v0.26.4
   Compiling zbus_names v2.6.1
   Compiling tempfile v3.10.1
   Compiling tokio-native-tls v0.3.1
   Compiling libpulse-binding v2.28.1
   Compiling async-executor v1.8.0
   Compiling zbus_macros v3.15.1
   Compiling async-broadcast v0.5.1
   Compiling enum_dispatch v0.3.12
   Compiling ogg v0.8.0
   Compiling serde_spanned v0.6.5
   Compiling maybe-async v0.2.10
   Compiling async-recursion v1.0.5
   Compiling zerocopy-derive v0.6.6
   Compiling dirs-sys-next v0.1.2
   Compiling clap_lex v0.7.0
   Compiling winnow v0.6.5
   Compiling numtoa v0.1.0
   Compiling strsim v0.11.0
   Compiling zerocopy v0.6.6
   Compiling clap_builder v4.5.1
   Compiling termion v1.5.6
   Compiling dirs-next v1.0.2
   Compiling toml_edit v0.22.6
   Compiling rspotify-http v0.12.0
   Compiling zbus v3.15.1
   Compiling rspotify-model v0.12.0
   Compiling libpulse-simple-binding v2.28.1
   Compiling lewton v0.10.2
   Compiling hyper-tls v0.5.0
   Compiling librespot-audio v0.4.2
   Compiling num v0.2.1
   Compiling zbus_names v3.0.0
   Compiling librespot-metadata v0.4.2
   Compiling zbus_macros v4.1.2
   Compiling async-broadcast v0.7.0
   Compiling rand_distr v0.4.3
   Compiling rustls-pemfile v1.0.4
   Compiling serde_urlencoded v0.7.1
   Compiling sha2 v0.10.8
   Compiling encoding_rs v0.8.33
   Compiling ipnet v2.9.0
   Compiling shell-words v1.1.0
   Compiling sync_wrapper v0.1.2
   Compiling ncspot v1.1.0 (/home/builder/.termux-build/ncspot-mpris/src)
   Compiling half v1.8.3
   Compiling rspotify-macros v0.12.0
   Compiling serde_cbor v0.11.2
   Compiling rspotify v0.12.0
   Compiling zbus v4.1.2
   Compiling reqwest v0.11.24
   Compiling librespot-playback v0.4.2
   Compiling parse_duration v2.1.1
   Compiling cursive v0.20.0
   Compiling notify-rust v4.10.0
   Compiling toml v0.8.10
   Compiling clap v4.5.1
   Compiling platform-dirs v0.3.0
   Compiling cursive_buffered_backend v0.6.1
   Compiling strum_macros v0.26.1
   Compiling arboard v3.3.2
error[E0433]: failed to resolve: could not find `Clipboard` in `platform`
  --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:76:38
   |
76 |         Ok(Clipboard { platform: platform::Clipboard::new()? })
   |                                            ^^^^^^^^^ could not find `Clipboard` in `platform`

error[E0433]: failed to resolve: could not find `Clear` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:156:31
    |
156 |         Clear { platform: platform::Clear::new(&mut self.platform) }
    |                                     ^^^^^ could not find `Clear` in `platform`

error[E0433]: failed to resolve: could not find `Get` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:161:29
    |
161 |         Get { platform: platform::Get::new(&mut self.platform) }
    |                                   ^^^ could not find `Get` in `platform`

error[E0433]: failed to resolve: could not find `Set` in `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:166:29
    |
166 |         Set { platform: platform::Set::new(&mut self.platform) }
    |                                   ^^^ could not find `Set` in `platform`

error[E0412]: cannot find type `Clipboard` in module `platform`
  --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:65:33
   |
65 |     pub(crate) platform: platform::Clipboard,
   |                                    ^^^^^^^^^ not found in `platform`

error[E0412]: cannot find type `Get` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:173:33
    |
173 |     pub(crate) platform: platform::Get<'clipboard>,
    |                                    ^^^ not found in `platform`

error[E0412]: cannot find type `Set` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:197:33
    |
197 |     pub(crate) platform: platform::Set<'clipboard>,
    |                                    ^^^ not found in `platform`

error[E0412]: cannot find type `Clear` in module `platform`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/arboard-3.3.2/src/lib.rs:238:33
    |
238 |     pub(crate) platform: platform::Clear<'clipboard>,
    |                                    ^^^^^ not found in `platform`

Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `arboard` (lib) due to 8 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `ncspot v1.1.0 (/home/builder/.termux-build/ncspot-mpris/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/ncspot-mpris/src/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-03-05 12:22:15 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/8155925825">8155925825</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/ncspot/build.sh
+++ b/packages/ncspot/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/hrkfdn/ncspot
 TERMUX_PKG_DESCRIPTION="An ncurses Spotify client written in Rust"
 TERMUX_PKG_LICENSE="BSD 2-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.0.0"
+TERMUX_PKG_VERSION="1.1.1"
 TERMUX_PKG_SRCURL=https://github.com/hrkfdn/ncspot/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=516663b62b9536cb18e6d8eb69470a5b6560f2890e010e8a3d2e8cfc65df9497
+TERMUX_PKG_SHA256=89fd70d625304ad882f2b3fcd048ea0e3910a7df371f942feded2ec57b4012b4
 TERMUX_PKG_DEPENDS="dbus, pulseaudio"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
@@ -13,7 +13,8 @@ TERMUX_PKG_CONFLICTS="ncspot-mpris"
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --no-default-features
---features cursive/termion-backend,share_clipboard,pulseaudio_backend
+--features termion_backend,pulseaudio_backend
 "
 # NOTE: ncurses-rs runs a test while building which fails while cross compiling:
-# therefore, we use cursive/termion-backend instead.
+# therefore, we use termion_backend instead.
+# share_clipboard cannot be used due to 1Password/arboard#56.

--- a/x11-packages/ncspot-mpris/build.sh
+++ b/x11-packages/ncspot-mpris/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/hrkfdn/ncspot
 TERMUX_PKG_DESCRIPTION="An ncurses Spotify client written in Rust (with MPRIS support)"
 TERMUX_PKG_LICENSE="BSD 2-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.0.0"
+TERMUX_PKG_VERSION="1.1.1"
 TERMUX_PKG_SRCURL=https://github.com/hrkfdn/ncspot/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=516663b62b9536cb18e6d8eb69470a5b6560f2890e010e8a3d2e8cfc65df9497
+TERMUX_PKG_SHA256=89fd70d625304ad882f2b3fcd048ea0e3910a7df371f942feded2ec57b4012b4
 TERMUX_PKG_DEPENDS="dbus, pulseaudio"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
@@ -13,7 +13,8 @@ TERMUX_PKG_CONFLICTS="ncspot"
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --no-default-features
---features cursive/termion-backend,share_clipboard,pulseaudio_backend,mpris,notify
+--features termion_backend,pulseaudio_backend,mpris,notify
 "
 # NOTE: ncurses-rs runs a test while building which fails while cross compiling:
-# therefore, we use cursive/termion-backend instead.
+# therefore, we use termion_backend instead.
+# share_clipboard cannot be used due to 1Password/arboard#56.

